.vscode/settings.json

```diff
@@ -1,5 +1,8 @@
 {
     "editor.formatOnSave": true,
+    "editor.codeActionsOnSave": {
```

Comment:
❓ just for my knowledge, what was the problem prior the change?

Reply:
When Olan try to edit the code on GitPod, the formatter wasn't been executed at save.